exocom-server/features/support/world.ls

```diff
@@ -116,7 +116,26 @@ World = !->
       expected.timestamp = that if received-message.timestamp
       expected.response-time = that if received-message.response-time
       expected.response-to = that if response-to
-      jsdiff-console received-message, expected, done
+      expect(received-message).to.include expected
+      expect(received-message.session-id).to.match /^\w|-*$/
+      done!
+
+
+  @verify-equal-session-id = ({client-name, message-name, id = '123', response-to}, done) ~>
+    service = @service-mocks[client-name]
+    wait-until (~> service.received-messages[0]?.name && service.received-messages.length == 2), ~>
+      service.received-messages.for-each (message) -> expect(message.session-id).to.match /^\w|-*$/
+      expect(service.received-messages[0].session-id).to.eql(service.received-messages[1].session-id)
+      done!
+
+
+  @verify-new-session-id = ({client-name, message-name, id = '123', response-to}, done) ~>
+    service = @service-mocks[client-name]
+    wait-until (~> service.received-messages[0]?.name && service.received-messages.length == 2), ~>
```

Comment:
Why `service.received-messages[0]?.name` here as well? Let's also make the conditional `service.received-messages.length > 1`